src/routes/About.jsx

```diff
@@ -49,9 +50,11 @@ function About() {
                     </div>
                     <div className='followUs-links'>
                         Follow us on:
-                        <img src={fLogo} alt='Facebook Logo' height={'35px'}></img>
-                        <img src={instagramLogo} alt='Instagram Logo' height={'35px'}></img>
-                        <img src={linkedinLogo} alt='LinkedIn Logo' height={'35px'}></img>
+                        <a href="#" target="_blank"><img src={fLogo} alt='Facebook Logo' height={'35px'}></img></a>
+                        <a href="https://www.instagram.com/microsoft.sist/" target="_blank"><img src={instagramLogo} alt='Instagram Logo' height={'35px'}></img></a>
+                        <a href="https://www.linkedin.com/company/microsoft-sist/" target="_blank"><img src={linkedinLogo} alt='LinkedIn Logo' height={'35px'}></img></a>
+                        <a href="https://github.com/Microsoft-Club-SIST" target="_blank"><img src={githublogo} alt="Github logo" height={'35px'}></img></a>
+                       
                     </div>
```

Comment:
Set `rel="noopener"` in `<a>` tag to ignore CI fails in deployment

Reply:
fixed 👍🏼